src/editor.c

```diff
@@ -510,16 +510,20 @@ e_context* e_initial(e_context* ctx, int c) {
       break;
     case 'v': {
       char* str = e_clipboard_paste();
-      e_context* new = e_context_copy(ctx);
-      new->history = ctx;
-      int i = 0;
-      while (str[i] != '\0') {
-        if (str[i] == '\n' || str[i] == '\r') e_insert_newline(new);
-        else e_insert_char(new, str[i]);
-        i++;
+      if (str != NULL){
```

Comment:
Instead you could write `if (!str) break;` as an early exit. That avoids wrapping everything in a big `if` statement.